### DISCUSSION TITLE
Can you provide android app demo

### DISCUSSION THREAD

**Discussion by hardik-foremost** [stasel/WebRTC-iOS#105] [General]
Hello sir...

Your IOS webSocket video app demo nice worked. I want to same p2p android side video calling using webSocket (wss) , can you possible to send him this demo app in android.

Thanks

**Comment by chanhdatng** [comment]
Did you find out any android samples? @hardik-foremost

**Comment by stasel** [comment]
Sounds like an awesome idea but unfortunately I'm not that good with Android nor have the capacity to work on such thing right now. Hopefully a skilled developer will use this repo to learn the concepts of WebRTC and apply them on Android. 
(The concept should be the same and the APIs should be similar)
